packages/virtualized-lists/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@react-native-mac/virtualized-lists",
-  "version": "0.1.0",
+  "version": "0.72.0-0",
```

Comment:
Do we want the "-0" or just keep it hardcoded to a specific version?

Reply:
Upstream seems to use caret: https://github.com/facebook/react-native/blob/f60c5ddc4baf545c65a299811b4f61ffee094c5d/packages/react-native/package.json#L104

Reply:
This is the package.json file for the virtualized-lists package itself. The caret belongs in (and is already in) packages/**react-native**/package.json.

Either way, the "-0" is just an arbitrary prerelease because beachball will (ideally) bump it to 0.72.0 once this gets checked in.

Reply:
Oh, my bad I thought this was a range. curious, why don't we do "0.72.2-rc0" or "-prerelease0"? Is one better than others?